### PR TITLE
Add article redirects

### PR DESCRIPTION
**Pull request by konradko**

### CODE REVIEW

File: ui/tests/test_url_redirects.py [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+import pytest
+
+from ui.urls import (
+    TOS_AND_PRIVACY_REDIRECT_LANGUAGES,
+    ARTICLE_REDIRECTS_MAPPING,
+    INTERNATIONAL_TRANSLATION_REDIRECTS_MAPPING
+)
+
+
+UTM_QUERY_PARAMS = '?utm_source=test%12&utm_medium=test&utm_campaign=test%test'
```
Comment (sdonk) [inline ui/tests/test_url_redirects.py]: We can use `urlencode` to make it more explicit (and Pythonic)

File: ui/tests/test_url_redirects.py [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+
+
+def add_utm_query_params(url):
+    return '{}{}'.format(url, UTM_QUERY_PARAMS)
```
Comment (sdonk) [inline ui/tests/test_url_redirects.py]: Can you please name the elements in the string? `{url}{params}.format(url=url, params=UTM_QUERY_PARAMS)`

File: ui/tests/test_url_redirects.py [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+
+international_url_and_param_values = []
+for url_code, param_value in INTERNATIONAL_TRANSLATION_REDIRECTS_MAPPING:
+    international_url_and_param_values.append(
+        ('/int/{}/'.format(url_code), param_value)
```
Comment (sdonk) [inline ui/tests/test_url_redirects.py]: same as before `{url}`

File: ui/tests/test_url_redirects.py [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+    url, expected_language, client
+):
+
+    if not url.endswith('/'):
+        url = client.get(url).url
+
+    response = client.get(url, follow=False)
+
+    assert response.status_code == http.client.MOVED_PERMANENTLY
+    assert response.url == '/international/?lang={}'.format(expected_language)
```
Comment (sdonk) [inline ui/tests/test_url_redirects.py]: ditto

File: ui/tests/test_url_redirects.py [hunk 5]
```diff
@@ -0,0 +1,152 @@
...
+
+    if not url.endswith('/'):
+        url = client.get(add_utm_query_params(url)).url
+    else:
+        url = add_utm_query_params(url)
+
+    response = client.get(url, follow=False)
+
+    assert response.status_code == http.client.MOVED_PERMANENTLY
+    assert response.url == '/international/{}&lang={}'.format(
```
Comment (sdonk) [inline ui/tests/test_url_redirects.py]: ditto

File: ui/urls.py [hunk 6]
```diff
@@ -541,3 +478,115 @@
...
+    ),
+    (
+        'write-an-export-plan',
+        'make-an-export-plan'
+    )
+)
+
+article_redirects = [
+    url(
+        r'^{}/$'.format(redirect[0]),
```
Comment (sdonk) [inline ui/urls.py]: ditto

File: ui/tests/test_url_redirects.py (hunk 2 above)
Comment (konradko) [reply ui/tests/test_url_redirects.py]: I find that useful in longer strings, but with short ones, it does not add to readability IMHO

**Pull request merged**